client/src/views/AuthorIndividualForm.vue

```diff
@@ -295,6 +326,7 @@ export default {
       }
 
       if (this.$v.$invalid || authorNameInvalid || authorCitizenshipDomicileInvalid) {
+        this.invalid = true
```

Comment:
There is a duplicate code block above that does the same thing

Reply:
I'll remove the one above it